tests/run_test.sh

```diff
@@ -239,6 +241,9 @@ fi
 
 if [[ $skip_check_results = false ]]; then
   check_results
+else
+  echo "Test ${TEST_NR} ${TEST_NAME} RUN_SUCCESS" >  ${REGRESSIONTEST_LOG}
```

Comment:
Do we need  to use ">>" since there might be several tests in the rt_weekly.conf? Also I am curious, do we know if the Test runs successfully here?

Reply:
The ${REGRESSIONTEST_LOG} here is specific to this individual test, each test will redirect it's output to a different log file. 
All we know at this moment is that job was successfully submitted and executed by scheduler, and that it didn't fail (i.e the executable didn't exit with non-zero return code).

Reply:
I think that we may also want to get running time info and memory info too, but that can be done in future PRs.